static/styles/audit.css

```diff
@@ -1,5 +1,6 @@
 @import url("pay.css");
 @import url("proxima.css");
+@import url("checkbox.css");
```

Comment:
I'm really not a fan of this CSS import approach, as it is supposed to be blocking and loading in series. Is there a better alternative?

Reply:
It would load one css file at the end due to bundling @pavlovcik

Reply:
Thank you for clarifying.